demisto_sdk/commands/common/constants.py

```diff
@@ -1423,3 +1423,17 @@ class IdSetKeys(Enum):
     FileType.DOC_IMAGE,
     FileType.TEST_PLAYBOOK,
 }
+
+
+class ParameterType(Enum):
```

Comment:
are you sure we dont have this somewhere already?

Reply:
Pretty much.. I moved this here from somewhere else, and nothing else came up in my search